snippets/css-snippets.cson

```diff
@@ -1,689 +1,707 @@
+    'body': 'background-repeat: ${1:repeat}'
   'background-repeat: repeat':
     'prefix': 'bgrr'
-    'body': 'background-repeat: repeat$0'
+    'body': 'background-repeat: repeat'
   'background-repeat: repeat-x':
     'prefix': 'bgrx'
-    'body': 'background-repeat: repeat-x$0'
+    'body': 'background-repeat: repeat-x'
   'background-repeat: repeat-y':
     'prefix': 'bgry'
-    'body': 'background-repeat: repeat-y$0'
+    'body': 'background-repeat: repeat-y'
   'background-repeat: no-repeat':
     'prefix': 'bgrn'
-    'body': 'background-repeat: no-repeat$0'
+    'body': 'background-repeat: no-repeat'
   'border':
     'prefix': 'bor'
-    'body': 'border: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}$0'
+    'body': 'border: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}'
   'border-color':
     'prefix': 'borc'
-    'body': 'border-color: ${1:#${2:000}}$0'
+    'body': 'border-color: ${1:#${2:000}}'
   'border-style':
     'prefix': 'bors'
-    'body': 'border-style: ${3:solid}$0'
+    'body': 'border-style: ${3:solid}'
   'border-width':
     'prefix': 'borw'
-    'body': 'border-width: ${1:${2:1}${3:px}}$0'
+    'body': 'border-width: ${1:${2:1}${3:px}}'
   'border-bottom':
     'prefix': 'borb'
-    'body': 'border-bottom: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}$0'
+    'body': 'border-bottom: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}'
   'border-bottom-width':
     'prefix': 'borbw'
-    'body': 'border-bottom-width: ${1:${2:1}${3:px}}}$0'
+    'body': 'border-bottom-width: ${1:${2:1}${3:px}}}'
   'border-left':
     'prefix': 'borl'
-    'body': 'border-left: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}$0'
+    'body': 'border-left: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}'
   'border-left-width':
     'prefix': 'borlw'
-    'body': 'border-left-width: ${1:${2:1}${3:px}}$0'
+    'body': 'border-left-width: ${1:${2:1}${3:px}}'
   'border-right':
     'prefix': 'borr'
-    'body': 'border-right: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}$0'
+    'body': 'border-right: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}'
   'border-right-width':
     'prefix': 'borrw'
-    'body': 'border-right-width: ${1:${2:1}${3:px}}$0'
+    'body': 'border-right-width: ${1:${2:1}${3:px}}'
   'border-top':
     'prefix': 'bort'
-    'body': 'border-top: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}$0'
+    'body': 'border-top: ${1:${2:1}${3:px}} ${3:solid} ${4:#${5:000}}'
   'border-top-width':
     'prefix': 'bortw'
-    'body': 'border-top-width: ${1:${2:1}${3:px}}$0'
+    'body': 'border-top-width: ${1:${2:1}${3:px}}'
   'border-radius':
     'prefix': 'br'
-    'body': 'border-radius: ${1:${2:1}${3:px}}$0'
+    'body': 'border-radius: ${1:${2:1}${3:px}}'
   'bottom':
     'prefix': 'bot'
-    'body': 'bottom: ${1:0}$0'
+    'body': 'bottom: ${1:0}'
+  'box-shadow':
+    'prefix': 'boxs'
+    'body': 'box-shadow: ${1:0px} ${2:3px} ${3:15px} ${4:1px} ${5:rgba(0,0,0,.5)};$0     // h-length, v-length, blur, spread, colour'
```

Comment:
1. Remove comment.  I keep my code under 80 characters, so this would make the line too long.  Maybe, move the comment above the snippet so it shows above the rule.
```scss
// comment
box-shadow: ...
```
\2. Remove $0 at end